websites/0-9/20XX/presence.ts

```diff
@@ -105,40 +105,43 @@ presence.on("UpdateData", async () => {
     }
 
     if (data20XX.user) {
-      data.state = `${data20XX.user.displayName} (${data20XX.user.isGuest ? 'guest' : `rank ${data20XX.user.rank}`})`;
+      data.state = `${data20XX.user.displayName} (${
+        data20XX.user.isGuest ? "guest" : `rank ${data20XX.user.rank}`
+      })`;
       if (data20XX.serverInfo)
         data.state += ` in ${data20XX.serverInfo.location}`;
     }
 
     if (data20XX.game) {
       data.details = `In-Game - ${data20XX.game.info.gametype} (${data20XX.game.info.players}/${data20XX.game.info.maxplayers})`;
 
-      if (!gameStartTimestamp)
-        gameStartTimestamp = Date.now();
-      
+      if (!gameStartTimestamp) gameStartTimestamp = Date.now();
+
       // Character
-      data.smallImageKey = 'char_' + data20XX.game.character.split('_')[0];
-      data.smallImageText = characterNameMap[data20XX.game.character.split('_')[0]];
+      data.smallImageKey = "char_" + data20XX.game.character.split("_")[0];
+      data.smallImageText =
+        characterNameMap[data20XX.game.character.split("_")[0]];
```

Comment:
but why

Reply:
this guy, just used a vsc pretty extension LMAO

Reply:
https://prettier.io/docs/en/rationale.html